recipes/parro/webview.js

```diff
@@ -0,0 +1,15 @@
+module.exports = Ferdium => {
```

Comment:
the lines that were generated for loading/importing the `service.css` are missing. please resurrect

Reply:
fixed